### ISSUE TITLE
Safe navigation operator ignored by `Layout/MultilineMethodCallIndentation`

### ISSUE THREAD

**Issue by deivid-rodriguez** [rubocop-hq/rubocop#5621]
## Expected behavior

I would expected the safe navigation operator to be considered by `Layout/MultilineMethodCallIndentation`.

## Actual behavior

The cop ignores method chaining when it uses `.&`.

### test.rb

```ruby
HakunaMatata.a
            .b
  &.c
  &.d

HakunaMatata.a
            .b
  .c
  .d
```

```bash
bundle exec rubocop --only Layout/MultilineMethodCallIndentation test.rb
```

only flags the second example.

## RuboCop version

```
$ rubocop -V
0.52.1 (using Parser 2.5.0.2, running on ruby 2.5.0 x86_64-linux)
```
